pyanaconda/modules/network/installation.py

```diff
@@ -186,6 +186,9 @@ def _write_interface_rename_config(self, root, ifname_option_values, overwrite):
 
         for ifname_value in ifname_option_values:
             iface, mac = ifname_value.split(":", 1)
+            # Avoid link files for 'nbft' interfaces as it breaks Boot from NVM over TCP
+            if iface.startswith('nbft'):
```

Comment:
Maybe we should not create the 'target_system_dir' few lines above if the list contains only "nbft*" values, so perhaps doing 
```
ifname_option_values = [value in ifname_option_values if not ifname_option_value.startswith('nbft')]
```
at the beginning of the method.